inc/extras.php

```diff
@@ -89,3 +89,13 @@ function twentyseventeen_is_frontpage() {
 		return false;
 	endif;
 }
+
+/**
+ * Add a pingback url auto-discovery header for singularly identifiable articles.
+ */
+function twentyseventeen_pingback_header() {
+	if ( is_singular() && pings_open() ) {
+		echo '<link rel="pingback" href="', bloginfo( 'pingback_url' ), '">' . "\n";
```

Comment:
Yeah, good call about the commas. I think this would be better:

`printf( '<link rel="pingback" href="%s">' . "\n", get_bloginfo( 'pingback_url' ) );`

That, the concatenation is more natural. And `bloginfo` outputs, it doesn’t return so this wouldn't echoing on an echo.

@HardeepAsrani Want to update the PR?